src/Model/APContact.php

```diff
@@ -368,10 +368,13 @@ public static function getByURL(string $url, $update = null): array
 
 		// Unhandled from Kroeg
 		// kroeg:blocks, updated
```

Comment:
What's this comment for?

Reply:
This is some old stuff that I added when I did my first steps with AP.

Reply:
Can you please remove it then?
```suggestion
```

Reply:
Done